Makefile

```diff
@@ -12,20 +12,22 @@ include Configfile
 
 PROJECT_NAME := $(shell basename $(CURDIR))
 
-# .PHONY: init\:
-# init::
-# 	@mkdir -p variables
-# ifndef GITHUB_USER
-# 	$(info GITHUB_USER not defined)
-# 	exit -1
-# endif
-# 	$(info Using GITHUB_USER=$(GITHUB_USER))
-# ifndef GITHUB_TOKEN
-# 	$(info GITHUB_TOKEN not defined)
-# 	exit -1
-# endif
-
-# -include $(shell curl -fso .build-harness -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3.raw" "https://raw.github.ibm.com/ICP-DevOps/build-harness/master/templates/Makefile.build-harness"; echo .build-harness)
+.PHONY: init\:
+init::
+	@mkdir -p variables
+ifndef GITHUB_USER
+	$(info GITHUB_USER not defined)
+	exit -1
+endif
+	$(info Using GITHUB_USER=$(GITHUB_USER))
+ifndef GITHUB_TOKEN
+	$(info GITHUB_TOKEN not defined)
+	exit -1
+endif
+
+ifdef BUILD_WITH_HARNESS
+-include $(shell curl -fso .build-harness -H "Authorization: token ${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3.raw" "https://raw.github.ibm.com/ICP-DevOps/build-harness/master/templates/Makefile.build-harness"; echo .build-harness)
```

Comment:
are we allowed to access `raw.github.ibm.com`?

Reply:
yes that works on github,ibm.com but not from github.com, that's why there is a condition.... when being on github.com the BUILD_WITH_HARNESS is not set.